### PR TITLE
Time.com.js -- fix eslint warnings

### PR DESCRIPTION
**Pull request by retorquere**

### CODE REVIEW

**Comment by retorquere** [comment]
updated for @zotero/eslint-config@1.0.2

File: Time.com.js [hunk 1]
```diff
@@ -9,34 +9,60 @@
...
 	"translatorType": 4,
 	"browserSupport": "gcsv",
-	"lastUpdated": "2017-05-23 21:37:25"
+	"lastUpdated": "2019-06-10 23:02:32"
 }
 
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017-2019 Michael Berkowitz
```
Comment (zuphilip) [inline Time.com.js]: Let us change the date to 2008 here.

**Comment by zuphilip** [comment]
This looks fine. Just change the date in the copyright notice and I merge it.

**Comment by retorquere** [comment]
I've changed the date.

**Comment by zuphilip** [comment]
Thank you!

**Pull request merged**